sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -57,6 +57,12 @@
       <version>1.2.1</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-jdk-httpclient</artifactId>
+      <version>1.0.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core-http-jdk-httpclient;dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
Having this as a general include will fail build on Java 10 and below, I think this dependency will need to be added through a profile to gate it on Java 11+.

@anuchandy @srnagar what did you do to work around this issue when building the repo in Java 8?

Reply:
had to override the profile inherited from the parent pom, like this https://github.com/Azure/azure-sdk-for-java/blob/cd464cfe7c217332e1d02a29b7a78c3ff04ef9b6/sdk/core/azure-core-http-jdk-httpclient/pom.xml#L131

Reply:
I have tried in parent pom. It triggers dependency cycle problem.

Reply:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=368008&view=logs&j=9d510ff3-53a8-5192-bd2d-466ce830bc13&t=01de8c6b-9f4b-5017-894e-951524cf562e